dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java

```diff
@@ -18,18 +19,31 @@
     private long id;
     private String entryType;
     private String versionName;
+    private DescriptorLanguage descriptorTypeInEnum = null;
```

Comment:
I don't think I like `descriptorTypeInEnum`; why not just `descriptorLanguage`? Although see my other comment for the PR.

Reply:
The naming is there to differentiate workflow type(which is an enum not a list) from tool's descriptor types(this is a list), and we could always change the names :)